tests/locationConfigTests.json

```diff
@@ -85,5 +85,16 @@
           "azureBlobSAS": "sv=2015-04-05&sr=b&si=tutorial-policy-635959936345100803&sig=9aCzs76n0E7y5BpEi2GvsSv433BZa22leDOZXX%2BXXIU%3D",
           "azureContainerName": "s3test"
         }
+    },
+    "gcp-test": {
```

Comment:
We can write up a document for this. We'll share it with Laure along with the jsdoc. If you have a specific format in mind, please let us know.

Reply:
It should be a .rst document in the /docs folder probably (I'll let Lauren confirm) so it can be integrated as part of our documentation. If you do a draft we can review together and suggest where to integrate it;